Svc/DpCatalog/test/ut/seq/Makefile

```diff
@@ -9,4 +9,4 @@ install: all
 	cp *.bin ../../../../../Ref/seq
```

Comment:
Delete the makefile.

Reply:
I will if you help me write a CMake one! Otherwise I'll keep it for my testing until I have time to learn CMake.